backend/routes/api.php

```diff
@@ -155,6 +155,7 @@
         ], function () {
             Route::get('/', 'TeamController@getTeam');
             Route::post('/', 'TeamController@inviteTeamMember');
+            Route::get('/relate', 'TeamController@getRelateUserWebsites');
```

Comment:
relate is verb, you cannot use verbs in routes. Even you are getting websites as I understood, so it should be resource /websites/me, or smth like this